### ISSUE TITLE
Create a `dev` branch for development activities

### ISSUE THREAD

**Issue by hkiyomaru** [ku-nlp/ja-vicuna-qa-benchmark#16]
## Why

Since our project is public, it is crucial to maintain the stability of the main branch.

## How

Create a development branch (`dev`). All development work will be directed to the branch, ensuring that the main branch remains stable at all times. Changes from the dev branch will be merged into the main branch only after thorough testing and review.

## Procedure

```bash
# Ensure you are on the main branch
git checkout main

# Pull the latest changes from the remote repository
git pull

# Create and switch to the new 'dev' branch
git checkout -b dev

# Push the 'dev' branch to the remote repository
git push -u origin dev
```
